src/pretix/control/middleware.py

```diff
@@ -41,7 +41,7 @@
 from django.template.response import TemplateResponse
 from django.urls import get_script_prefix, resolve, reverse
 from django.utils.encoding import force_str
-from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
I don't get how this is related to the issue and it might not be the correct fix. The call to `_()` down below in the file is not executed at import time (which is why we usually need gettext_lazy), but only on view time.

I believe the reason we are seeing flaky translation on the error message is a different one: `LocaleMiddleware`, which decides which language to use for the request, is run **after** `PermissoinMiddleware`, so we'd need to swap the order of these two middlewares.

Reply:
Sorry, removed.